### PR TITLE
[12.0][FIX] agreement_operating_unit

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Fixed UT.

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 Can you please review this PR?

**Review by max3903 — APPROVED** [review]

**Review by Chandresh-SerpentCS — APPROVED** [review]
LGTM.

File: agreement_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "summary": """
         This module adds operating unit information to agreements and service
         profiles.""",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.1.0",
```
Comment (dreispt) [inline agreement_operating_unit/__manifest__.py:9]: Ocabot takes care of this, without the risk of code conflicts, so not needed.

**Review by dreispt — APPROVED** [review]

**Pull request merged**